hello.js

```diff
@@ -0,0 +1,6 @@
+let x = Myfunction(4, 3);
+
+function Myfunction(A, B) {
+// Function returns the product of a and b
+  return A * B;
```

Comment:
Change the variable name to small letter

Reply:
changes are done please check